build.sbt

```diff
@@ -180,7 +180,7 @@ lazy val root = (project in file("."))
     libraryDependencies ++= (coreDependencies ++ testDependencies),
     mainClass := Some("org.constellation.ConstellationNode")
     // other settings here
-  )
+  ).aggregate(wallet, keytool)
```

Comment:
Why? It's not needed IMO

Reply:
Oh that just automatically makes jars for all sub-projects in the aggregate args, I'm lazy and didn't wanna compile them all individually. Should we remove for any reason?

Reply:
Re: dm about slow compile time, will remove